client/src/js/app.js

```diff
@@ -553,10 +553,20 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
 
   /* Patient record */
   .state('patientRecord', {
+    abstract : true,
     url : '/patients/:patientID',
-    controller: 'PatientRecordController as PatientRecordCtrl',
-    templateUrl: 'partials/patients/record/patient_record.html'
+    templateUrl: 'partials/patients/record/patient_record.html',
+    controller: 'PatientRecordController as PatientRecordCtrl'
   })
+    .state('patientRecord.details', {
+      url : '',
+      views : {
+        'checkin@patientRecord' : { 
```

Comment:
I was playing around with absolute vs. relative names to see how specific these routes could get - as this is a fairly simple heirachy this will work just using the `checkin` namespace.